tests/Unit/Evolution/Systems/Ccz4/Test_TimeDerivative.cpp

```diff
@@ -0,0 +1,1244 @@
+  const std::array<double, 3> spin{{0.3, 0.5, 0.2}};
+  const std::array<double, 3> center{{0.2, 0.3, 0.4}};
+  const gr::Solutions::KerrSchild solution(mass, spin, center);
+
+  // Setup grid
+  const size_t num_points_1d = 6;
+  const std::array<double, 3> lower_bound{{0.8, 1.22, 1.30}};
+  const std::array<double, 3> upper_bound{{0.82, 1.24, 1.32}};
+  Mesh<SpatialDim> mesh{num_points_1d, Spectral::Basis::Legendre,
+                        Spectral::Quadrature::GaussLobatto};
+  const auto coord_map =
+      domain::make_coordinate_map<Frame::ElementLogical, FrameType>(Affine3D{
+          Affine{-1., 1., lower_bound[0], upper_bound[0]},
+          Affine{-1., 1., lower_bound[1], upper_bound[1]},
+          Affine{-1., 1., lower_bound[2], upper_bound[2]},
+      });
+  const size_t num_points_3d = num_points_1d * num_points_1d * num_points_1d;
+  const DataVector used_for_size =
+      DataVector(num_points_3d, std::numeric_limits<double>::signaling_NaN());
+  // Setup coordinates
+  const auto x_logical = logical_coordinates(mesh);
+  const auto x = coord_map(x_logical);
+  // Arbitrary time for time-independent solution.
+  const double t = std::numeric_limits<double>::signaling_NaN();
+  // Evaliuate analytic solution
+  const auto kerrschild_vars = solution.variables(
+      x, t, typename gr::Solutions::KerrSchild::tags<DataVector>{});
+
+  // Get ingredients for computing arguments to Ccz4::TimeDerivative
+  const auto& spatial_metric =
+      get<gr::Tags::SpatialMetric<SpatialDim, FrameType, DataVector>>(
+          kerrschild_vars);
+  const auto& d_spatial_metric =
+      get<Tags::deriv<gr::Tags::SpatialMetric<SpatialDim>,
+                      tmpl::size_t<SpatialDim>, FrameType>>(kerrschild_vars);
+  const auto det_and_inverse = determinant_and_inverse(spatial_metric);
+  const auto det_spatial_metric = det_and_inverse.first;
+  const auto d_det_spatial_metric =
+      get<gr::Tags::DerivDetSpatialMetric<SpatialDim, FrameType>>(
+          solution.variables(
+              x, t,
+              tmpl::list<
+                  gr::Tags::DerivDetSpatialMetric<SpatialDim, FrameType>>{}));
+  const auto& dt_spatial_metric =
+      get<Tags::dt<gr::Tags::SpatialMetric<SpatialDim>>>(kerrschild_vars);
+  const auto& inverse_spatial_metric =
+      get<gr::Tags::InverseSpatialMetric<SpatialDim, FrameType, DataVector>>(
+          kerrschild_vars);
+  const auto& lapse = get<gr::Tags::Lapse<DataVector>>(kerrschild_vars);
+  const auto& d_lapse =
+      get<Tags::deriv<gr::Tags::Lapse<DataVector>, tmpl::size_t<SpatialDim>,
+                      FrameType>>(kerrschild_vars);
+  const auto& shift =
+      get<gr::Tags::Shift<SpatialDim, FrameType, DataVector>>(kerrschild_vars);
+  const auto& d_shift =
+      get<Tags::deriv<gr::Tags::Shift<SpatialDim, FrameType, DataVector>,
+                      tmpl::size_t<SpatialDim>, FrameType>>(kerrschild_vars);
+
+  // Params
+  const double c = 1.0;
+  const double cleaning_speed = 1.6;  // e
```

Comment:
`e` -> `this is e in the paper`? Here, given only one latin letter is a bit tricky to read. I think `\tau` is fine :)

Reply:
Yes, I can extend the comment